apps/nsq_to_nsq/nsq_to_nsq.go

```diff
@@ -31,10 +31,8 @@ const (
 
 var (
 	showVersion = flag.Bool("version", false, "print version string")
-
-	topic       = flag.String("topic", "", "nsq topic")
 	channel     = flag.String("channel", "nsq_to_nsq", "nsq channel")
-	destTopic   = flag.String("destination-topic", "", "destination nsq topic")
+	singleDestinationTopic   = flag.String("single-destination-topic", "", "activate single destination nsq topic Note: default destination topic is the topic itself")
```

Comment:
If the user wants to send different topics to different nsqds, they should run multiple instances of nsq_to_nsq.

(That's also the already-possible way to achieve what this PR seeks to make more convenient - multiple instances.)